Eevee.Sleep.Bot/Modules/SlashCommands/RoleManagementSlashModule.cs

```diff
@@ -83,6 +83,72 @@ public class RoleManagementSlashModule : InteractionModuleBase<SocketInteraction
         );
     }
 
+    [SlashCommand("add-all", "Add all owned tracked roles.")]
+    [UsedImplicitly]
+    public async Task AddAllRoleAsync() {
+        var user =
+            Context.User as SocketGuildUser ??
+            throw new InvalidOperationException("User is not SocketGuildUser.");
+
+        var previousRoleIds = DiscordTrackedRoleContoller
+            .FindAllTrackedRoleIdsByRoleIds(user.Roles.Select(x => x.Id).ToArray())
+            .Select(x => x.Id)
+            .ToArray();
+
+        await user.AddRolesAsync(DiscordRoleRecordController.FindRoleIdsByUserId(user.Id));
+
+        await Context.Interaction.RespondAsync(
+            embed: DiscordMessageMaker.MakeChangeRoleResult(
+                user: user,
+                previousRoleIds: previousRoleIds,
+                currentRoleIds: DiscordRoleRecordController.FindRoleIdsByUserId(user.Id),
+                color: Colors.Success
+            ),
+            ephemeral: true
+        );
+    }
+
+    [SlashCommand("remove-all", "Remove all tracked roles.")]
+    [UsedImplicitly]
+    public async Task RemoveAllRoleAsync() {
+        var user =
+            Context.User as SocketGuildUser ??
+            throw new InvalidOperationException("User is not SocketGuildUser.");
+
+        var previousRoleIds = DiscordTrackedRoleContoller
+            .FindAllTrackedRoleIdsByRoleIds(user.Roles.Select(x => x.Id).ToArray())
+            .Select(x => x.Id)
+            .ToArray();
+
+        await user.RemoveRolesAsync(DiscordRoleRecordController.FindRoleIdsByUserId(user.Id));
+
+        await Context.Interaction.RespondAsync(
+            embed: DiscordMessageMaker.MakeChangeRoleResult(
+                user: user,
+                previousRoleIds: previousRoleIds,
+                currentRoleIds: [],
+                color: Colors.Danger
+            ),
+            ephemeral: true
+        );
+    }
+
+    [SlashCommand("show", "Shows all the owned tracked roles.")]
+    [UsedImplicitly]
+    public Task ShowRoleAsync() {
+        var user =
```

Comment:
Refactor all `Context.User as SocketGuildUser ?? ...` calls to `DiscordExtension.AsGuildUser()` then call it like `Context.User.AsGuildUser()`.